.circleci/config.yml

```diff
@@ -274,6 +282,32 @@ jobs:
           path: fastlane/test_output/xctest
           destination: scan-test-output
 
+  compile-xcode-13-2:
+    <<: *base-job
+    steps:
+      - setup-git-credentials
+      - restore_cache:
+          keys:
+            - homebrew-tap-cache-3
+      - run:
+          name: Install swiftlint 0.48.0
+          command: |
+              brew update --preinstall
+              brew tap-new $USER/local-tap
+              brew extract --version=0.48.0 swiftlint $USER/local-tap
+              brew install swiftlint@0.48.0
+      - save_cache:
+          key: homebrew-tap-cache-3
+          paths:
+            - /usr/local/Homebrew/Library/Taps/$USER/homebrew-local-tap/
+            - /Users/$USER/Library/Caches/Homebrew/
+      - checkout
+      - install-bundle-dependencies
+      - run:
+          name: SPM Build
+          command: swift build
```

Comment:
TODO: replace with `pod lib lint` instead

Reply:
Why not using `pod lib lint` now?

Reply:
I wrote this last night, I'm doing it now hehe

Reply:
Going back to `swift build` because `pod lib lint` fails with the old Xcode:
> Undefined symbols for architecture arm64:
      "_swift_getFunctionTypeMetadataGlobalActorBackDeploy", referenced from:
          type metadata accessor for @Swift.MainActor @Sendable (RevenueCat.StoreTransaction?, RevenueCat.CustomerInfo?, __C.NSError?, Swift.Bool) -> () in CustomerInfo.o
          type metadata accessor for @Swift.MainActor @Sendable () -> () in Deprecations.o
    ld: symbol(s) not found for architecture arm64